src/BSD-2-Clause-NetBSD.xml

```diff
@@ -0,0 +1,42 @@
+<SPDX name="BSD 2-clause NetBSD License" identifier="BSD-2-Clause-NetBSD" osi-approved="false">
+  <urls>
+    <url>http://www.netbsd.org/about/redistribution.html#default</url>
+  </urls>
+  <notes>NetBSD adopted this 2-clause license in 2008 for code contributed to NetBSD Foundation.</notes>
+  <license>
+    <copyright>
+      <p>Copyright (c) 2008 The NetBSD Foundation, Inc. All rights reserved.</p>
+    </copyright>
+    <body>
+      <p>This code is derived from software contributed to The NetBSD Foundation by 
```

Comment:
should this line be optional?  I think we decided this made it a different license b/c of this, but... might want to have another look?